package.json

```
@@ -3,9 +3,9 @@
   "type": "module",
   "version": "0.0.1",
   "scripts": {
-    "dev": "partykit dev & astro dev",
+    "dev": "partykit dev && astro dev",
```

We'll sort this out in the Windows issue that will get created. Below works on non-Windows environments.

```suggestion
    "dev": "partykit dev & astro dev",
```